charts/datadog-monitor/values.yaml

```diff
@@ -0,0 +1,66 @@
+global:
+  # Add labels from a parent chart to all manifests.
+  labels: {}
+
+fullnameOverride: ""
+##################################################
+#                 Datadog Monitors               #
+##################################################
+# https://github.com/DataDog/datadog-operator/tree/main/examples/datadogmonitor
+
+# The name of the your project. This will be prefixed on all DataDog monitors along with service name.
```

Comment:
```
The name of the your project. This will be prefixed on all DataDog monitors along with service name.
Eg. - name: norge-vask will look like [norge-vask]-capture-service-some-monitor-name
```

The correct term here is `release name` instead of `service name`.